src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/aggregation/TableAggregation.java

```diff
@@ -0,0 +1,128 @@
+/*-
+ * #%L
+ * OBKV Table Client Framework
+ * %%
+ * Copyright (C) 2021 OceanBase
+ * %%
+ * OBKV Table Client Framework is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at:
+ *          http://license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ * See the Mulan PSL v2 for more details.
+ * #L%
+ */
+
+package com.alipay.oceanbase.rpc.protocol.payload.impl.execute.aggregation;
+
+import com.alipay.oceanbase.rpc.ObClusterTableQuery;
+import com.alipay.oceanbase.rpc.filter.ObTableFilter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TableAggregation {
+    public TableAggregation(ObClusterTableQuery obClusterTableQuery) {
+        this.tablequery_ = obClusterTableQuery;
+    }
+
+    public void Min(String columnName) {
```

Comment:
I guess "Min" is a C++ style. Instead, Camel-Case is our choose.

Reply:
fixed